packages/swap/src/routes/v2/utils.ts

```diff
@@ -95,7 +94,7 @@ export const getLatestSwapForId = async (id: string) => {
       where: { nativeId: BigInt(id) },
       include: swapRequestInclude,
     });
-  } else if (txHashRegex.test(id)) {
```

Comment:
solana transaction signatures are base58 encoded. we could also adjust the regex for that

Reply:
oof forgot about polkadot

Reply:
okay i thought about this and decided to not use a regex here for now. the regex would save us one db request if somebody requests a status for something that is not a valid ref. i think that is negligible if it happens by mistake. and if somebody wants to spam us intentionally, he probably is sophisticated enough to use something that looks like a ref

Reply:
the upside of not having the regex is that we dont need to think about adjusting it if something changes (and we also cannot fuck up the regex lol)

Reply:
Yeah, i think i would be fine with opening this fallback so we don't need to handle each case separately and possibly save us some time in the future as well!